wiki/content/query-language/fragments.md

```diff
@@ -6,10 +6,14 @@ weight = 25
     parent = "query-language"
 +++
 
-`fragment` keyword allows you to define new fragments that can be referenced in a query, as per [GraphQL specification](https://facebook.github.io/graphql/#sec-Language.Fragments). The point is that if there are multiple parts which query the same set of fields, you can define a fragment and refer to it multiple times instead. Fragments can be nested inside fragments, but no cycles are allowed. Here is one contrived example.
+The `fragment` keyword lets you to define new fragments that can be referenced
+in a query, per the [GraphQL specification](https://facebook.github.io/graphql/#sec-Language.Fragments).
+Fragments allow for the reuse of common repeated selections of fields, reducing
+duplicated text in the DQL documents. Fragments can be nested inside fragments,
+but no cycles are allowed in such cases. For example:
```

Comment:
I would advise possible the term looping for clarity of what a cycle is being referred to:

>but no looping cycles are allowed

Reply:
Thank you, very useful. This was the sentence that I was unsure about.